src/app/containers/ArticleTimestamp/shouldLastUpdatedTimestampBeShown.js

```diff
@@ -0,0 +1,33 @@
+import { isLastRelative, isSameDay, isToday } from './helpers';
```

Comment:
Can this be renamed to `shouldDisplayLastUpdated` or something along those lines?

Reply:
👍